lib/kubernetes-deploy/kubernetes_resource/deployment.rb

```diff
@@ -4,6 +4,7 @@
 module KubernetesDeploy
   class Deployment < KubernetesResource
```

Comment:
This class and probably others have optimizations like `@latest_rs = exists? ? find_latest_rs(cache) : nil` that will no longer take effect. Please check all of them and either (a) remove them if they're useless now; or (b) remove the resource in question from the prewarming list (probably only for Node on DS, but I'm not sure).